instrumentation/otel-webserver-module/src/apache/ApacheTracing.cpp

```diff
@@ -47,7 +47,7 @@ void ApacheTracing::writeTrace(server_rec* s, const char* funcName, const char*
 
     if (m_state == TRACE_AS_ERROR && s)
     {
-        ap_log_error(APLOG_MARK, APLOG_ERR, 0, s, "mod_appdynamics: %s: %s", funcName, note);
+        ap_log_error(APLOG_MARK, APLOG_ERR, 0, s, "mod_opentelemetry_wbserver_sdk: %s: %s", funcName, note);
```

Comment:
Thanks for raising this.
Spelling mistake ?
wbserver ---> webserver

Reply:
Yes, needs to read my patch one more time ...